src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/email/ReportSender.java

```diff
@@ -31,16 +35,11 @@
     public ReportSender(
         JavaMailSender mailSender,
         ReportsService reportsService,
-        String[] recipients
+        @Value("${reports.recipients}") String reportRecipientsCsv
     ) {
         this.mailSender = mailSender;
         this.reportsService = reportsService;
-
-        if (recipients == null) {
-            this.recipients = new String[0];
-        } else {
-            this.recipients = Arrays.copyOf(recipients, recipients.length);
-        }
+        this.recipients = reportRecipientsCsv.split(",");
```

Comment:
does spring not support lists in config?
I'm pretty sure we've used it somewhere already. List of jurisdictions I think.

Reply:
it's the way you store list of recipients. since they have to be populated in vault and members of list vary this can be configured as comma separated values. especially when content is as we have now

Reply:
list of jurisdictions was stored the same way.
see here https://github.com/hmcts/bulk-scan-processor/pull/482

Reply:
i see what you mean now. automatically parse into array of strings as spring will do the split for you